packages/mds-db/geographies.ts

```diff
@@ -10,35 +10,53 @@ import { getReadOnlyClient, getWriteableClient } from './client'
 import { ReadGeographiesParams, PublishGeographiesParams } from './types'
 
 export async function readSingleGeography(geography_id: UUID): Promise<Geography> {
-  try {
-    const client = await getReadOnlyClient()
+  const client = await getReadOnlyClient()
 
-    const sql = `select * from ${schema.TABLE.geographies} where geography_id = '${geography_id}'`
```

Comment:
I believe we are trying to maintain the use of the sqlVals from the sql-utils library throughout our codebase.